libs/components/tabs/src/lib/modules/tabs/tabset.component.ts

```diff
@@ -199,6 +186,8 @@ export class SkyTabsetComponent implements AfterViewInit, OnDestroy {
 
   public tabButtons: TabButtonViewModel[] = [];
 
+  public elementRole: string | undefined = 'tablist';
```

Comment:
Seems like `'tablist'` is the default element role? If so, I would save this value in a const at the top of the page to use it both places. `const DEFAULT_ELEMENT_ROLE = 'tablist';`